src/components/Icon/Icon.md

```diff
@@ -0,0 +1,29 @@
+#### Default:
+```jsx
+<Icon name="IconAlertTriangle" color="highlight" size={20} stroke={2} />
+```
+
+#### Thicker Icon:
+
+```js
+
+<Icon name="IconAlertTriangle" color="highlight" size={20} stroke={4} />
+
+```
+
+#### Primary Icon:
+
+```js
+
+<Icon name="IconAlertTriangle" color="highlight" size={20} stroke={2} />
+
+```
+
+
+#### Large Icon:
+
+```js
+
+<Icon name="IconAlertTriangle" color="highlight" size={24} stroke={2} />
+
+```
```

Comment:
Would be good to document the component a bit better, explain where the user can see all the available icon names, how they look and so on. We need to make this documentation more like a documentation and not just code examples

Reply:
Yeah, that makes sense, I'm going to add a better doc on it